src/controllers/update/update.review.managing.officer.corporate.controller.ts

```diff
@@ -27,7 +27,13 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     if (appData?.managing_officers_corporate){
       dataToReview = appData?.managing_officers_corporate[Number(index)];
       principalAddress = (dataToReview) ? mapDataObjectToFields(dataToReview[PrincipalAddressKey], PrincipalAddressKeys, AddressKeys) : {};
-      serviceAddress = (dataToReview) ? mapDataObjectToFields(dataToReview[ServiceAddressKey], ServiceAddressKeys, AddressKeys) : {};
+      let undefinedPrincipalAddressProperties;
```

Comment:
is the code across the 3 files exactly the same? Might be better extracting this out into a method to call from the 3 places?

Reply:
Yeah it is the same, will do